R/lvl0_loadSources.R

```diff
@@ -122,6 +122,7 @@ lvl0_loadEU <- function(input_folder, EU_dir = "EU_data"){
   ## road passenger: load annual mileage
   am_countries_EU<- do.call("rbind",lapply(list_countries$countries,
                                            function(x) {
+                                             browser()
```

Comment:
It is preferable to use a interactive procedure to select chunks to be selected for commits. If you use git bash, you could do `git add -pu` to avoid such leftovers.